docs/why-ngrok.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Why ngrok?
+---
+
+## Environment Independence
+
+ngrok works everywhere. But more than that, it works _the same_ everywhere.
+
+You can run your services in AWS, GCP or Azure. You can run services in your
+on-prem datacenters or on IoT devices like a Raspberry Pi. You can run your
+services on hosted container platforms like Heroku, Render or Fly. You can even
```

Comment:
services on hosted container platforms like Heroku, Render, or Fly. You can even